src/containers/Masthead/components/MastheadSearchForm.jsx

```diff
@@ -101,6 +105,22 @@ export class MastheadSearchForm extends Component {
     }
   }
 
+  async handleTaxonomyId(taxId) {
+    const { history } = this.props;
+    let taxonomyFunction = fetchTopic;
+    if (taxId.includes('urn:resource:')) {
+      taxonomyFunction = fetchResource;
+    }
+    try {
+      const taxElement = await taxonomyFunction(taxId);
+      const arr = taxElement.contentUri.split(':');
+      const id = arr[arr.length - 1];
+      history.push(toEditArticle(id, 'standard'));
```

Comment:
`toEditArticle` tar også `locale` men den er optional, vet ikke om man skal redirecte til en språk basert edit side, spørs på use-casen

Reply:
I dette tilfellet har du ikkje formeining om språk. toEditArticle uten språk plukker et av dei tilgjengelige språka for visning.